CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.23.3]
+### Changed
```

Comment:
Nit: there is a `Removed` category for removals. We don't use it often, but it seems appropriate here

```suggestion
### Removed
```